zenodo/config.py

```diff
@@ -181,6 +181,17 @@ def _(x):
     ),
 )
 
+# SIPStore
+# ========
+#: Default JSON schema for the SIP agent
+SIPSTORE_DEFAULT_AGENT_JSONSCHEMA = 'sipstore/agent-webclient-v1.0.0.json'
+
+#: Enable the agent JSON schema
+SIPSTORE_AGENT_JSONSCHEMA_ENABLED = True
+
+#: Enable the agent JSON schema
+SIPSTORE_FILEPATH_MAX_LEN = 1000
```

Comment:
This value: https://github.com/inveniosoftware/invenio-files-rest/blob/master/invenio_files_rest/config.py#L65
has not been overwritten in Zenodo anywhere. I guess this one can stay like this for now ?